test/osb_test/catalog_test.go

```diff
@@ -429,7 +430,12 @@ type prefixedBrokerHandler struct {
 
 func (pbh *prefixedBrokerHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	if strings.HasPrefix(req.URL.Path, pbh.brokerPathPrefix) {
-		common.SetResponse(w, http.StatusOK, common.Object{"services": common.Array{}})
+		servicesMap := make(map[string]interface{})
+		err := json.Unmarshal([]byte(simpleCatalog), &servicesMap)
```

Comment:
Why is this necessary?

Reply:
the current test registers a broker with empty catalog and expects a notification to be created in the DB, while with the new code no notifications are created in this case - there are no plans to notify any platforms about. 
In order to keep the test for broker credentials change while there is a notification in the DB i needed to adapt it so a notification will be created by adding the catalog content.